src/Fantomas.Core/MultipleDefineCombinations.fs

```diff
@@ -35,8 +35,28 @@ type CodeFragment =
         | Content(lineCount = lineCount) -> lineCount
         | NoContent _ -> 0
 
-    override this.Equals _ = false
-    override this.GetHashCode() = Int32.MinValue
+    override x.Equals(other: obj) =
+        match other with
+        | :? CodeFragment as other ->
+            match x, other with
+            | HashLine _ as x, (HashLine _ as other) -> x = other
+            | Content _ as x, (Content _ as other) -> x = other
+            | NoContent _ as x, (NoContent _ as other) -> x = other
+            | _ -> false
+        | _ -> false
+
+    override x.GetHashCode() =
```

Comment:
@dawedawe I didn't really know how to best deal with this one.
https://learn.microsoft.com/en-us/dotnet/api/system.hashcode.combine?view=net-7.0 doesn't seem to be a `netstandard2.0` thing.

Let me know if you have any other suggestions.

Reply:
No, that seems really good enough for me 👍